### DISCUSSION TITLE
SendPhoto whith MarkdownV2, cant send newline in caption

### DISCUSSION THREAD

**Discussion by Kris24333** [mymmrac/telego#97] [Q&A] [answered]
Hello

I use this code
```
text := "Some text \n\n Some text"
if _, err := telegramBot.Bot.SendPhoto(&telego.SendPhotoParams{
			ChatID:      tu.ID(update.Message.Chat.ID),
			ParseMode:   parseMode,
			Caption:     text,
			ReplyMarkup: keyboard,
			Photo:       tu.File(mustOpen(photoPath)),
		}); err != nil {
			log.Println(err)
		}
```
Newline dont work 

debug:
`API call to: "https://api.telegram.org/botBOT_TOKEN/sendPhoto", with data: parameters: {"caption":"Some text\\n\\nSome text","chat_id":"122345","parse_mode":"MarkdownV2"...` - extra \ in Api call

**Comment by mymmrac** [comment, marked as answer]
Thanks for noticing this, there actually was a serious bug that was preventing any strings to be properly sent when files were attached.

Fixed this in [v0.22.1](https://github.com/mymmrac/telego/releases/tag/v0.22.1), please try with new version

**Reply by Kris24333** [reply]
Thank you :)
